src/framework/uicomponents/view/popupview.cpp

```diff
@@ -30,13 +31,142 @@
 
 #include "log.h"
 
+namespace mu::uicomponents {

```

Comment:
I'd suggest to move this class into ../internal

Reply:
I'm not sure, it does not have independent value, it's just part of the implementation of `PopupView`